packages/material-ui/src/Fade/Fade.js

```diff
@@ -74,7 +75,7 @@ Fade.propTypes = {
   /**
    * A single child content element.
    */
-  children: PropTypes.element,
+  children: elementAcceptingRef,
```

Comment:
A different possible? path: https://github.com/mui-org/material-ui/issues/15515#issuecomment-487663610.

Reply:
Probably not. We need the host instance. It doesn't break any test because the instance is a requirement.

Reply:
From what I understand, the `Fade` component doesn't need to access the host instance. We are only forwarding the ref for convenience and consistency. It's a different answer for Slide that needs it.

Reply:
Hmm, they all need to forward refs to work inside other components e.g. Modal

Reply:
I haven't considered this point. So the strategy is to warn early and potentially have false positive: #15515 to improve the DX. ThIs makes sense

Reply:
I'm commenting on #15515 with a solution for formik.

Reply:
Makes sense. Fade, Grow and Zoom don't actually need the ref. Will revert the warning there and we can apply @oliviertassinari's patch in a different PR.